hack/tools/release/notes/list_test.go

```diff
@@ -65,3 +65,134 @@ func Test_buildSetOfPRNumbers(t *testing.T) {
 		})
 	}
 }
+
+func Test_githubFromToPRLister_listPRs(t *testing.T) {
+	tests := []struct {
+		name    string
+		fields  *githubFromToPRLister
+		args    ref
+		wantErr bool
+	}{
+		{
+			name: "Successful PR Listing",
+			fields: &githubFromToPRLister{
+				client: &githubClient{
+					repo: "kubernetes-sigs/kind",
```

Comment:
How about using kubernetes-sigs/cluster-api ?
Probably, to use kind repository doesn't affect in the future, but to use cluster-api is more safety since the repo is itself.